src/org/labkey/test/util/query/QueryUtils.java

```diff
@@ -0,0 +1,42 @@
+package org.labkey.test.util.query;
+
+import org.labkey.remoteapi.CommandException;
+import org.labkey.remoteapi.Connection;
+import org.labkey.remoteapi.query.DeleteRowsCommand;
+import org.labkey.remoteapi.query.SelectRowsCommand;
+import org.labkey.remoteapi.query.SelectRowsResponse;
+import org.labkey.remoteapi.query.TruncateTableCommand;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.util.TestLogger;
+
+import java.io.IOException;
+
+public class QueryUtils
+{
+    private QueryUtils()
+    {
+        // Prevent instantiation
+    }
+
+    public static void truncateTable(String containerPath, String schema, String table) throws IOException, CommandException
+    {
+        Connection cn = WebTestHelper.getRemoteApiConnection();
+        TruncateTableCommand cmd = new TruncateTableCommand(schema, table);
+        cmd.execute(cn, containerPath);
+    }
+
+    public static void selectAndDeleteRows(String containerPath, String schema, String table) throws IOException, CommandException
```

Comment:
Maybe change the title to selectAndDelete**All**Rows?

Also do you  think a comment in this class might be useful to help people understand when they should call truncateTable and when they should call selectAndDelete?

Reply:
Good idea.